.dockerignore

```diff
@@ -0,0 +1,5 @@
+.terraform
+.terraform.lock.hcl
+.terraform.tfstate.backup
+.terraform.tfstate.lock.info
+terraform.tfvars
```

Comment:
With these - is it worth going further and excluding all of the terraform files (`*.tf`) from your Docker image? Would you ever want them there?

Reply:
added *.tf to .dockerignore